python/tchannel/tornado/stream.py

```diff
@@ -118,6 +124,12 @@ def write(self, chunk):
             self._condition.notify()
```

Comment:
Why doesn't `write` fail with an exception if the stream has failed with an exception